plutus-core/plutus-core/src/PlutusCore/Crypto/ModExp.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE CPP         #-}
+{-# LANGUAGE MagicHash   #-}
+{-# LANGUAGE UnboxedSums #-}
+module PlutusCore.Crypto.ModExp
+    ( modularExponentiation
+    ) where
+
+import PlutusCore.Builtin
+
+import GHC.Natural
+
+#ifdef VERSION_ghc_bignum
```

Comment:
This doesn't seem to work.  I've got GHC 9.6.6 but if I try to run any script involving `modularExponentiation` I get errors like this:
```
An error has occurred:
The machine terminated because of an error, either from a built-in function or from an explicit use of 'error'.
Caused by: modularExponentiation 9 2 17
modularExponentiation: FIXME: stub for GHC8.10, report to plutus developers
```
If I change `ifdef` to `ifndef` then it works.